index.html

```diff
@@ -280,6 +280,18 @@ <h3 id="aidno_verticalhrefno_verticalnoverticala"><a id="no_vertical" href="#no_
 <li><code>book:tag</code> - <a href="#string">string</a> <a href="#array">array</a> -
 Tag words associated with this book.</li>
 </ul>
+<p><a name="type_payment" href="#type_payment"><code>payment</code></a> - Namespace URI: <a href="https://ogp.me/ns/payment"><code>https://ogp.me/ns/payment#</code></a><version>Beta only</version></p>
```

Comment:
1. This content should get generated from the markdown (see #69 for how once those are in sync), sot the "Beta only" bit there will look different.
2. `<version>` isn't a valid HTML element, so this isn't going to work anyway. I think just using what gets generated from the markdown is probably fine for now.